### DISCUSSION TITLE
We just released a fresh new listening experience for Spreaker

### DISCUSSION THREAD

**Discussion by lele85** [livewire/livewire#8059] [2. Show and tell]
Hey there! Spreaker is one of the leading podcast hosting monetization and distribution companies, and i'm happy to announce that we just released the first iteration of our new web listening experience.

https://next.spreaker.com

![next spreaker com_](https://github.com/livewire/livewire/assets/49144/04d27d05-644b-4525-a4d0-4b1b33d24c50)

All done with:

- Laravel
- Livewire + AlpineJS
- BrefPHP on AWS lambda

The development experience was absolutely sweet and is mainly due to the great tools you guys have built (and trust me when i say that because i'm usually a nodejs + react person).

That's it. If you have questions, feedbacks or anything else just ask and i will try to respond to all of them!

**Comment by joshhanley** [comment]
@lele85 nice work! Thanks for sharing 🤩
